frontend/web/components/FeatureValue.tsx

```diff
@@ -23,22 +23,25 @@ const FeatureValue: FC<FeatureValueType> = (props) => {
   }
   const isCompact = getViewMode() === 'compact'
   return (
-    <span
-      className={classNames(`chip ${props.className || ''}`, {
-        'chip--sm justify-content-start d-inline': isCompact,
+    <div
+      className={classNames(`chip flex-row ${props.className || ''}`, {
+        'chip--sm justify-content-start': isCompact,
       })}
       onClick={props.onClick}
       data-test={props['data-test']}
+      style={{ maxWidth: 'fit-content' }}
     >
-      {type == 'string' && <span className='quot'>"</span>}
-      <span className='feature-value'>
-        {Format.truncateText(
-          `${Utils.getTypedValue(props.value)}`,
-          isCompact ? 24 : 20,
-        )}
+      <span style={{ overflow: 'hidden', textOverflow: 'ellipsis' }}>
+        {type == 'string' && <span className='quot'>"</span>}
+        <span className='feature-value'>
+          {Format.truncateText(
+            `${Utils.getTypedValue(props.value)}`,
+            isCompact ? 24 : 20,
+          )}
+        </span>
+        {type == 'string' && <span className='quot'>"</span>}
```

Comment:
It looks like this end quote gets ellipsed too when there is no separator in the value.
I think that moving the overflow in the feature-value span along with a `flexWrap: nowrap` in the div and removing the extra-span could fix it (and maybe the little width gap we see in between the 2 ellipsed values in the screenshot).

Something like
```
<div ... >
    {type == 'string' && <span className='quot'>"</span>}
    <span className='feature-value' style:{{ ellipsis }}>
      ....
      </span>
    {type == 'string' && <span className='quot'>"</span>}
```

Wdyt ?